### PR TITLE
Refactor/Client/Refactor HouseInfo(rule) page

### PR DESCRIPTION
**Pull request by hyejin-seok**
## Describe your changes

- Rename to House Info page (from Rules page)
- Add a list of Members
- Add a message for the case with no rules

## Issue
No. 7

No. | Page/API | BE/FE | Error details
-- | -- | -- | -- 
7 | /house-info (/rules) | FE | Now only the owner can see the member in the house, but it's better to show in somewhere the member list for all members?→ Below Rule, show member list

## URL
http://localhost:3000/house-info

## Screenshot (if you made the UI)
Screenshot 2024-04-15 at 14 46 50
Screenshot 2024-04-15 at 14 47 20

- **When there is no rule**

Screenshot 2024-04-12 at 16 19 03

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings and no errors

### CODE REVIEW

File: client/app/house-info/component/HouseInfoList.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        )}
+      </div>
+      <h2 className='mt-6 text-lg font-semibold md:mt-8 md:text-2xl'>
+        Members
+      </h2>
+      <div className='mt-2 flex flex-col gap-3 rounded-md border border-amber-400 bg-amber-100 px-2 py-4 md:mt-4 md:px-7 md:py-7'>
+        {members.map((member) => (
+          <div key={member.id} className='flex items-center'>
+            <FiUser className='mr-2 h-auto w-5 min-w-5 text-teal-700 md:mr-3 md:w-6 md:min-w-6' />
+            <p className='text-sm font-semibold md:text-lg'>{member.name}</p>
```
Comment (kanatagu) [inline client/app/house-info/component/HouseInfoList.tsx:35]: I think span is a more proper element

File: client/app/house-info/component/HouseInfoList.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          </p>
+        )}
+      </div>
+      <h2 className='mt-6 text-lg font-semibold md:mt-8 md:text-2xl'>
+        Members
+      </h2>
+      <div className='mt-2 flex flex-col gap-3 rounded-md border border-amber-400 bg-amber-100 px-2 py-4 md:mt-4 md:px-7 md:py-7'>
+        {members.map((member) => (
+          <div key={member.id} className='flex items-center'>
+            <FiUser className='mr-2 h-auto w-5 min-w-5 text-teal-700 md:mr-3 md:w-6 md:min-w-6' />
```
Comment (kanatagu) [inline client/app/house-info/component/HouseInfoList.tsx:34]: Oh, could you show the colour icon like the member list?

File: client/app/page.tsx [hunk 3]
```diff
@@ -24,8 +24,12 @@ export default function Home() {
...
             title={houseName}
             buttonComponent={
-              <Button variant={'outline'} onClick={() => router.push('/rules')}>
-                Rules
+              <Button
+                variant={'outline'}
+                className='ml-3'
+                onClick={() => router.push('/house-info')}
+              >
+                House Info
```
Comment (kanatagu) [inline client/app/page.tsx:32]: Better to add `align-Items:center` for the title and the right button!
Screenshot 2024-04-15 at 1 45 22 PM

File: client/app/house-info/component/HouseInfoList.tsx (hunk 1 above)
Comment (hyejin-seok) [reply client/app/house-info/component/HouseInfoList.tsx]: Fixed it. 
a63ea2419ad78feef6d96d5103709ac0b61a5e77

File: client/app/house-info/component/HouseInfoList.tsx (hunk 2 above)
Comment (hyejin-seok) [reply client/app/house-info/component/HouseInfoList.tsx]: Fixed it.
8df63529543e211e671446e296f1608caaca816f

File: client/app/page.tsx (hunk 3 above)
Comment (hyejin-seok) [reply client/app/page.tsx:32]: Added it.
553dcf24bb4ebd09d9f85ed231d6e37376a6d457

File: client/app/house-info/component/HouseInfoList.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+        )}
+      </div>
+      <h2 className='mt-6 text-lg font-semibold md:mt-8 md:text-2xl'>
+        Members
+      </h2>
+      <div className='mt-2 flex flex-col gap-3 rounded-md border border-amber-400 bg-amber-100 px-2 py-4 md:mt-4 md:px-7 md:py-7'>
+        {members.map((member) => (
+          <div key={member.id} className='flex items-center'>
+            <MdAccountCircle
+              className={`mr-2 h-auto w-5 min-w-5 md:mr-3 md:w-7 md:min-w-7 ${colorMap[member.icon.toString()]}`}
```
Comment (kanatagu) [inline client/app/house-info/component/HouseInfoList.tsx:38]: I feel the icon is too small; maybe it's better like `w-7 md:w-8`!

**Review by kanatagu — APPROVED** [review]
LGTM!

File: client/app/house-info/component/HouseInfoList.tsx (hunk 4 above)
Comment (hyejin-seok) [reply client/app/house-info/component/HouseInfoList.tsx]: Fixed it. 
911849df26771841faff69c58ac715e5f9dd5167

**Pull request merged**